libs/components/modals/src/lib/modules/modal/modal-host.component.ts

```diff
@@ -57,8 +58,10 @@ export class SkyModalHostComponent implements OnDestroy {
   #injector: Injector;
   #router: Router;
   #changeDetector: ChangeDetectorRef;
-
   #modalHostContext: SkyModalHostContext;
+  #elRef: ElementRef;
+
+  #ariaPreviousValueSiblings = new Map<Element, string | null>();
```

Comment:
Since the `SkyModalAdapterService` is provided by this component, its state will be tied to this component as well. Meaning, the adapter service could manage all of the sibling elements itself. This would also keep the modal component from being "polluted" with DOM features and types (the adapter service is designed to abstract away the DOM-specific aspects of our component class).

If possible, I would move the `#ariaPreviousValueSiblings` property to the adapter service and have it manage the `aria-hidden` states and elements itself. The only thing you should be calling in this component would be:
```
this.#adapter.hideHostSiblingsForScreenReaders(this.#elRef);
this.#adapter.showHostSiblingsForScreenReaders(this.#elRef);

this.#adapter.hidePreviousModalForScreenReaders();
this.#adapter.showPreviousModalForScreenReaders();
```